api/db/seeds/data/team-certification/data-builder.js

```diff
@@ -765,7 +765,7 @@ async function _createAPublishedV3CertificationSession({ databaseBuilder }) {
     });
 
     const { id: lastAssessmentResultId } = databaseBuilder.factory.buildAssessmentResult({
-      juryId: 9000,
+      juryId: TEAM_CERTIFICATION_OFFSET_ID_USERS,
```

Comment:
Je propose de passer sur REAL_PIX_SUPER_ADMIN_ID ; le juryId étant l'id d'un utilisateur Pix Admin (qu'importe lequel) qui laisse un commentaire à destination interne.

```suggestion
      juryId: REAL_PIX_SUPER_ADMIN_ID,
```

La constante utilisée dans cette modification est une constante genraliste, dont SCO_CERTIFICATION_MANAGING_STUDENTS_ORGANIZATION_USER_ID est un alias, et qui est + un user destiné à Pix Orga de ce que je crois comprendre du code.

Reply:
@HEYGUL j'ai modifié du coup suite à notre conversation à l'oral :)